sips/re_registering_attack_protection.md

```diff
@@ -0,0 +1,95 @@
+| Author                      | Title                           | Category | Status              |
+|-----------------------------| ------------------------------- | -------- | ------------------- |
+| Lior Rutenberg (@lior-blox) | SSV Register Cluster copied shares attack | Core     | open-for-discussion |
+
+## Summary
+
+A "copied shares" attack can occur when a malicious actor registers valid shares to a legitimate validator by either "frontrunning" a valid registration of a cluster transaction, or re-registering a previously removed validator.
+
+In both scenarios, a validator would be operating on the SSV network under the control of a harmful user instead of the rightful validator owner.
+
+#### Front-Running
+In this scenario, a malicious actor could "front-run" a genuine user trying to register a validator. The registration would appear as legitimate. To keep the attack ongoing, the attacker would cover the validator's fees.
+
+To make any changes (such as removing or changing the cluster), the genuine user would need to exit the validator.
+
+#### Re-registering attack
```

Comment:
add also removing from a different cluster section explaining it

Reply:
If you want to use crypto lingo, then you can call it a "replay attack" or registration replay.
Just a NIT

Reply:
@lior-blox please fix

Reply:
fixed it